Small bug fix as ssbj_mdf.py script was not working on latest OpenMDA…

imcovangent opened the pull request:
…O commit -> ScipyOptimizer() changed to ScipyOptimizeDriver().

Commit:
Small bug fix as ssbj_mdf.py script was not working on latest OpenMDAO commit -> ScipyOptimizer() changed to ScipyOptimizeDriver().

relf commented on ssbj_mdf.py:
```diff
@@ -13,7 +13,7 @@
 from openmdao.api import Problem
 from openmdao.api import SqliteRecorder, WebRecorder
-from openmdao.api import ScipyOptimizer #, pyOptSparseDriver
+from openmdao.drivers.scipy_optimizer import ScipyOptimizeDriver
```
Thank you Imco. Normally, ScipyOptimizer should still work as it is only deprecated, except that [a bug was recently fixed](https://github.com/OpenMDAO/OpenMDAO/commit/abab34e0adae3e4acd36803b6d09e97c0cf0fa10#diff-e952a3a8f61dba125351945b8a2086bf) on that topic. But we can move on and indeed use the new name ScipyOptimizerDriver. Just one small thing: you could modify the pull request to keep using <code>openmdao.api</code> which is the OpenMDAO way to import stuff. To do that, just commit again, push and the pull request will be updated automatically.

Commit:
Small code clean-up in ssbj_mda.py
Updated optimizer to ScipyOptimizeDriver using import via api
Fixed small bug in name == main code of structure.py
